import-export-cli/integration/testutils/app_testUtils.go

```diff
@@ -71,6 +71,7 @@ func exportApp(t *testing.T, name string, owner string, env string) (string, err
 
 	t.Cleanup(func() {
 		base.RemoveApplicationArchive(t, getEnvAppExportPath(env), name, owner)
+		base.WaitForIndexing()
```

Comment:
No need to wait here because we are only removing the export archive file and nothing related to indexing is taking place in APIM side.